dev-infrastructure/modules/acr-push-permission.bicep

```diff
@@ -0,0 +1,19 @@
+param principalName string
```

Comment:
can we refactor the acr-push-permission and acr-pull-permission modules into one and and parameterize on the permission? e.g. have a look at the keyvault-secret-access module